src/components/Molecules/OnboardingButton/OnboardingButton.interface.ts

```diff
@@ -2,7 +2,7 @@ import * as H from 'history'
 import { ButtonProps } from '@/components/Atoms/Button'
 
 export interface OnboardingButtonProps extends ButtonProps {
-  onboardingClick: (ev: React.MouseEvent<HTMLElement>) => void
+  onboardingClick?: (ev: React.MouseEvent<HTMLElement>) => void
```

Comment:
This button for the feature requires a click handler 
The `to:?` can stay as it's definitely optional :)

```suggestion
  onboardingClick: (ev: React.MouseEvent<HTMLElement>) => void
```